controllers/apply/awards-for-all/fields.js

```diff
@@ -1541,7 +1542,11 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             explanation: localise({
                 en: `We’ll use this whenever we get in touch about the project`,
                 cy: `Byddwn yn defnyddio hwn pan fyddwn yn cysylltu ynglŷn â’r prosiect`
-            })
+            }),
+            schema: Joi.string()
```

Comment:
If we wanted to we could probably push the `.lowercase()` into the base `EmailField` class to avoid duplicating but this is OK for now.